### DISCUSSION TITLE
Auto-Format for Thumbnails?

### DISCUSSION THREAD

**Discussion by niklasgrewe** [directus/directus#15796] [Archive]
Hi, thank you for the dynamic way to generate thumbnails, its easy and i love it. 👍 The only thing i miss is `format=auto`. Many cloud providers like cloudinary already support this. When a request is sent, the `accept` header is checked for supported formats and the most efficient format is selected, mostly webp or now avif.

I would be very happy if directus would support this too. This would also avoid some frontend code like `<source />` tags etc.

Let me know what you think about it

**Comment by azrikahar** [comment]
This is discussed in #12626, particularly the comments over here: https://github.com/directus/directus/discussions/13626#discussioncomment-2856699 😉 Feel free to upvote it!
